src/nlp/parsetext.ts

```diff
@@ -127,21 +128,19 @@ export default function parseText (text: string, language: Language = ENGLISH) {
       // DAILY on weekdays is not a valid rule
       case 'weekday(s)':
         options.freq = RRule.WEEKLY
-        options.byweekday = [
-          RRule.MO,
-          RRule.TU,
-          RRule.WE,
-          RRule.TH,
-          RRule.FR
-        ]
+        options.byweekday = [RRule.MO, RRule.TU, RRule.WE, RRule.TH, RRule.FR]
         ttr.nextSymbol()
+        AT()
+        COLON()
         F()
         break
 
       case 'week(s)':
         options.freq = RRule.WEEKLY
         if (ttr.nextSymbol()) {
           ON()
+          AT()
+          COLON()
```

Comment:
The main draw back with this approach is that lists of times are not supported though that may be intentional.

For example in the list case you would have to list all permutation for this to work
```
// Invalid
every day at 9:00 and 11:30 => RRULE:FREQ=DAILY;BYHOUR=9,11;BYMINUTE=0,30
// valid
every day at 9:00, 9:30, 11:00 and 11:30 => RRULE:FREQ=DAILY;BYHOUR=9,11;BYMINUTE=0,30
```

Personally, I think it's reasonable to limit this to one hour time combo to prevent confusion